modules/reactor/test/tests/meshgenerators/polygon_concentric_circle_mesh_generator/pin_hex_reporting_id.i

```diff
@@ -0,0 +1,52 @@
+[Mesh]
+  [pin]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '6 6 6 6 6 6'
```

Comment:
We should ease up on the discretization for all these golded meshes. Mesh files should be as slim as possible if we want to be sustainable

Reply:
I will reduce the golded mesh file size.